setup/ubuntu14.04/ansible-vars.yaml

```diff
@@ -15,4 +15,3 @@ packages:
   - libtool
   - curl
   - ntp
-  - ccache
```

Comment:
@jbergstroem are we doing ccache from source or putting up with what comes in by default?